.travis.yml

```diff
@@ -1,90 +1,52 @@
 language: python
-# if the https://travis-ci.org/ODM2/ODM2PythonAPI/requests ever says: missing config
```

Comment:
There was a lot of copy-n-paste stuff here from other projects that I don't think they are really need for `yodatools`. However, if I removed something that is needed, please let me know!